modules/user_manual/pages/files/webgui/sharing.adoc

```diff
@@ -86,6 +86,8 @@ permissions applied, except for the ability to share with other users.
 
 image:guest-users/content-shared-with-guest-user.png[Content shared with a guest user]
 
+NOTE: Guest users can also use the **desktop sync client** to keep their files up to date.
```

Comment:
```suggestion
NOTE: Guest users can also use the **Desktop/IOS/Andriod Sync Clients** to access and/or sync files locally.
```

Reply:
I don't think you can access your files without synching them. By connecting your account to the server your are synching the server state to your device. I think we can leave out the and/or. 

What do you think?

Reply:
good point, I have updated the suggestion. thanks!